CHANGELOG.md

```diff
@@ -4,6 +4,73 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+
+## 2.17.0 (2024-09-20)
+
+This release adds support for PostgreSQL 17, significantly improves the performance of continuous aggregate refreshes,
+and contains performance improvements for analytical queries and delete operations over compressed hypertables.
+We recommend that you upgrade at the next available opportunity.
+
+**Highlighted features in TimescaleDB v2.17.0**
+
+* Full PostgreSQL 17 support for all existing features. TimescaleDB v2.17 is available for PostgreSQL 14, 15, 16 and 17.
+
+  PostgreSQL 17 support for the Toolkit Extension will follow soon and will be announced on [Toolkit's release notes](https://github.com/timescale/timescaledb-toolkit/releases).
+
+* Significant performance improvements for continuous aggregate policies: Continuous aggregate refresh is now using
+`merge` instead of deleting old materialized data and re-inserting.
+
+  This update can decrease dramatically the amount of data that must be written on the continuous aggregate in the
+  presence of a small number of changes, reduce the `i/o` cost of refreshing a continuous aggregate and generate less
+  `wal`.
```

Comment:
Your call, but both I/O and [WAL](https://www.postgresql.org/docs/current/wal-intro.html) are canonically referenced in capital letters

```suggestion
  presence of a small number of changes, reduce the `I/O` cost of refreshing a continuous aggregate and generate less
  amount of Write-Ahead Logs (`WAL`).
```

Reply:
Edited so.